tdrs-frontend/nginx/local/locations.conf

```diff
@@ -4,9 +4,10 @@ location = /nginx_status {
     deny all;
 }
 
-location ~ ^/(v1|admin|static/admin|static/drf-yasg|swagger|redocs) {
+location ~* ^/(v1|admin|static/admin|static/drf-yasg|swagger|redocs)/(.*)$ {
```

Comment:
why did we have to do this change exactly? the existing regex should capture all locations with trailing URI sections

Reply:
good call. i saw this change in the original pr and thought it was needed, but i tested locally and seems to work without it. removed.